package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -145,18 +145,13 @@
 
 logger = logging.getLogger("MDAnalysis.coordinates.AMBER")
 
+from scipy.io import netcdf
```

Comment:
I am not a big fan of renaming modules to arbitrary names. I could just do
```python
import scipy.io.netcdf
```
and use the full module name everywhere, which would be very clear. `netCDF4` is the proper name so I'd want to keep it.